packages/codelabs/src/basics/web-components.md

```diff
@@ -564,9 +479,17 @@ For example we can change the front of our page and it will affect the text insi
 
 Web Components are being used in the wild by many companies and in many projects. Some examples:
 
+### YouTube
+
+YouTube is one of the most popular sites on the internet. Their main website is built with custom elements to split up the different parts of their page in components. Additionally, they are using Polymer behaviors to work with attributes and properties.
```

Comment:
```suggestion
YouTube is one of the most popular sites on the internet. Their main website is built with custom elements to split up the different parts of their page in components.
```

Reply:
No mentioning Polymer? ;-)

Reply:
I think in the context its more important to mention they're using custom elements, rather than specifically mention Polymer. People new to web components may not know what polymer is and exactly how it relates to custom elements

Reply:
Yes in the end, these behaviors are just mixins , tight to Custom Elements (which I already mention anyway)

Reply:
Yeah I really like that mention youtube, I was thinking about adding this the other day. 

I don't thinking mentioning polymer behaviors adds anything to the case here. Plus they're just using the polymer base class, right?